index.html

```diff
@@ -27,6 +28,7 @@
             </a>
         </div>
      </nav>
+    </header>    
     <header>
         <section>
             <div class="container">
```

Comment:
- [OPTIONAL] Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise. It also means that you should not create a new branch for re-review changes you should introduce the changes in the same branch for the first review. Please apply this in your future projects.